corfudb-tools/README

```diff
@@ -45,15 +45,15 @@ org.corfudb.browser.CorfuBrowserMain
 Keystore and truststore parameters are not required if tls is disabled.
 
 For usage help,
-java -cp "corfudb-tools-0.4.2.1-SNAPSHOT-shaded.jar" org.corfudb.browser.CorfuStoreBrowserMain <--help|-h>
+java -cp "corfudb-tools-0.0.0.0-SNAPSHOT-shaded.jar" org.corfudb.browser.CorfuStoreBrowserMain <--help|-h>
```

Comment:
ditto - just curious why 0.0.0.0 instead of 4.2.2?

Reply:
Replied above.